samza-sql/src/main/java/org/apache/samza/sql/runner/SamzaSqlApplicationConfig.java

```diff
@@ -202,9 +209,16 @@ private UdfResolver createUdfResolver(Map<String, String> config) {
 
     Collection<RelRoot> relRoots = dslConverter.convertDsl(String.join("\n", dslStmts));
 
-    for (RelRoot relRoot : relRoots) {
-      SamzaSqlApplicationConfig.populateSystemStreams(relRoot.project(), inputSystemStreams, outputSystemStreams);
-    }
+    // the snippet below dose not work when sql is a query
```

Comment:
Please either do a // TODO or // FIXME, or remove this code.

Reply:
Thanks for reviewing this PR. 
"FIXME" has been added.